MMR.Randomizer/Resources/SEQS.txt

```diff
@@ -1,319 +1,331 @@
 mm-terminafield
-0
+0,102
```

Comment:
What are all these additions for?

Reply:
Each songslot now has its own matching category, which is just slot slot + 0x100.

This provides the means to allow music makers and users to select specific songs they want a song to be able to occupy without using general categories. 

Example 1: Maybe you have good ambient/quiet music that fits caves really well, but you don't think it suits all dungeons, you can give it category 13B to specify so that it doesn't end up in the louder/livelier dungeon music areas.

Example 2: If you have music that best fits ice/cold areas, you can now specify you want that music to be possible to be used in mountains/goronshrine/snowhead without using broad field/city/dungeon categories.